_infra/helm/secure-message-v2/templates/deployment.yaml

```diff
@@ -134,5 +134,28 @@ spec:
             {{- end }}
           - name: GOOGLE_APPLICATION_CREDENTIALS
             value: /var/secrets/google/credentials.json
+          - name: UAA_CHECK_ENABLED
+            value: true
```

Comment:
I think as this is a keyword boolean for the Go templating it needs to be quoted as `"true"`. It's not technically a boolean in the context of the environment variable it's assigned to. Once exposed as an environment variable string Python will assign it and use it as a boolean.
```
json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string
```